frontend/__tests__/components/features/home/task-suggestions.test.tsx

```diff
@@ -9,8 +9,12 @@ import { TaskSuggestions } from "#/components/features/home/tasks/task-suggestio
 import { SuggestionsService } from "#/api/suggestions-service/suggestions-service.api";
 import { MOCK_TASKS } from "#/mocks/task-suggestions-handlers";
 import { AuthProvider } from "#/context/auth-context";
+import { ProviderOptions } from "#/types/settings";
 
-const renderTaskSuggestions = (initialProvidersAreSet = true) => {
+const renderTaskSuggestions = (
+  initialProvidersAreSet = true,
+  githubProviderSet = true,
```

Comment:
This should work for now, but it seems like this is just coding around the assumption that github and gitlab are the only two providers, rather than generalizing. For purposes of a hotfix that's probably fine.

Reply:
Yeah good point